kentikapi/client.go

```diff
@@ -89,15 +83,14 @@ type RetryConfig = httputil.RetryConfig
 func NewClient(c Config) (*Client, error) {
 	c.FillDefaults()
 
-	syntheticsConnection, err := makeConnForGRPC(c, c.SyntheticsHostPort)
-	if err != nil {
-		return nil, fmt.Errorf("grpc synthetics connection: %v", err)
+	if err := parseAPIURL(&c.APIURL); err != nil {
+		return nil, fmt.Errorf("invalid api url: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("make API v5 URL: %v", err)
```
Error message should describe what the just-called function does. "invalid API URL: %v" message or "parse API URL: %v" would be appropriate next to url.Parse() call.
See also: https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
Done